docs/guide/getting-started.md

```diff
@@ -120,3 +120,11 @@ class PagesController < ApplicationController
   end
 end
 ```
+
+### Rendering ViewComponents outside of the view context
+
+To render ViewComponents outside of the view context (such as in a background job, markdown processor, etc), instantiate a Rails controller:
```

Comment:
A Rails controller or a ViewComponent? The code sample instantiates an `ApplicationComponent` 🤔

Reply:
Oops! will fix.